### PR TITLE
Add rocketmq configuration items. Add orderly message. 

### PR DESCRIPTION
**Pull request by jinjianfeng-chn**
Add rocketmq configuration items. Add orderly message. Change the version of rocketmq-client-go to v2.1.0.

Signed-off-by: jinjianfeng-chn <feng081212@163.com>

# Description

_Please explain the changes you've made_

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -40,7 +40,7 @@ require (
 	github.com/aliyun/aliyun-tablestore-go-sdk v1.7.7
 	github.com/apache/dubbo-go-hessian2 v1.11.1
 	github.com/apache/pulsar-client-go v0.9.0
-	github.com/apache/rocketmq-client-go/v2 v2.1.1-rc2
+	github.com/apache/rocketmq-client-go/v2 v2.1.0
```
Comment (berndverst) [inline go.mod:43]: Instead can you please increase the version to `v2.1.1` and address any required changes?

I don't think going backwards is a good idea.

**Review by berndverst — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (jinjianfeng-chn) [reply go.mod:43]: v2.1.1 is unavailable due to compatibility problems

Comment (jinjianfeng-chn) [reply go.mod:43]: I am submitting code to the RocketMQ community to address compatibility issues.

Comment (berndverst) [reply go.mod:43]: Yes does not seem they maintained semantic versioning at all. Breaking change between 2.10 and 2.1.1, and even between 2.1.1-rc2 and 2.1.1. Very strange!

**Review by berndverst — APPROVED** [review]

**Comment by berndverst** [comment]
@jinjianfeng-chn please create a PR for the documentation updates (`v1.10` branch)

**Comment by jinjianfeng-chn** [comment]
> @jinjianfeng-chn please create a PR for the documentation updates (`v1.10` branch)

OK

**Pull request merged**